targets/esp8266/readme.md

```diff
@@ -13,38 +13,18 @@ You can view online from
 Follow [this](./docs/ESP-PREREQUISITES.md) page to setup build environment
 
 
-#### 2. Patch ESP-SDK for JerryScript
-
-Follow [this](./docs/ESP-PATCHFORJERRYSCRIPT.md) page to patch for JerryScript building.
-Below is a summary after SDK patch is applied.
-
-#### 3. Building JerryScript
+#### 2. Building JerryScript
 
 ```
-cd ~/harmony/jerryscript
-# clean build
-make -f ./targets/esp8266/Makefile.esp8266 clean
-# or just normal build
+# assume you are in jerryscript folder
 make -f ./targets/esp8266/Makefile.esp8266
 ```
 
 Output files should be placed at $BIN_PATH
 
-#### 4. Flashing for ESP8266 ESP-01 board (WiFi Module)
-
-Steps are for `ESP8266 ESP-01(WiFi)` board. Others may vary.
-Refer http://www.esp8266.com/wiki/doku.php?id=esp8266-module-family page.
-
-##### 4.1 GPIO0 and GPIO2
-
-Before flashing you need to follow the steps.
-
-1. Power off ESP8266
-2. Connect GPIO0 to GND and GPIO2 to VCC
-3. Power on ESP8266
-4. Flash
-
-##### 4.2 Flashing
+#### 3. Flashing for ESP8266 12E
```

Comment:
the previous readme was for `ESP8266 ESP-01`, now this one states `ESP8266 12E`. is the previous board not supported anymore?

Reply:
Unfortunately it is not supported anymore. The toolchain generates two binaries:
- eagle.flash.bin ~50kB
- eagle.irom0text.bin ~ 400 kB

The ESP8266 ESP-01 has only 512 kB flash memory. Unluckily you cannot put these images next to each other (in this case it would fit into 512 kB flash size), because you cannot write your user data before 0x20000. It means you start flashing your user binary at around 131kB and 131 + 400 > 512.